code/datums/supply_packs/food.dm

```diff
@@ -55,11 +55,140 @@
 					/obj/item/reagent_container/food/snacks/grown/tomato,
 					/obj/item/reagent_container/food/snacks/grown/tomato,
 					)
-	cost = 10
+	cost = 7
 	containertype = /obj/structure/closet/crate/freezer
 	containername = "\improper Tomato crate"
 	group = "Food"
 
+/datum/supply_packs/soybeans
+	name = "soybeans (x20)"
+	contains = list(
+					/obj/item/reagent_container/food/snacks/grown/soybeans,
+					/obj/item/reagent_container/food/snacks/grown/soybeans,
+					/obj/item/reagent_container/food/snacks/grown/soybeans,
+					/obj/item/reagent_container/food/snacks/grown/soybeans,
+					/obj/item/reagent_container/food/snacks/grown/soybeans,
+					/obj/item/reagent_container/food/snacks/grown/soybeans,
+					/obj/item/reagent_container/food/snacks/grown/soybeans,
+					/obj/item/reagent_container/food/snacks/grown/soybeans,
+					/obj/item/reagent_container/food/snacks/grown/soybeans,
+					/obj/item/reagent_container/food/snacks/grown/soybeans,
+					/obj/item/reagent_container/food/snacks/grown/soybeans,
+					/obj/item/reagent_container/food/snacks/grown/soybeans,
+					/obj/item/reagent_container/food/snacks/grown/soybeans,
+					/obj/item/reagent_container/food/snacks/grown/soybeans,
+					/obj/item/reagent_container/food/snacks/grown/soybeans,
+					/obj/item/reagent_container/food/snacks/grown/soybeans,
+					/obj/item/reagent_container/food/snacks/grown/soybeans,
+					/obj/item/reagent_container/food/snacks/grown/soybeans,
+					/obj/item/reagent_container/food/snacks/grown/soybeans,
+					/obj/item/reagent_container/food/snacks/grown/soybeans,
+					)
+	cost = 7
+	containertype = /obj/structure/closet/crate/freezer
+	containername = "\improper soybeans crate"
+	group = "Food"
+
+/datum/supply_packs/soybeans
+	name = "cocoapod,sugarcane (x10)"
```

Comment:
in all written languages that use the Latin alphabet, a space is required after a comma.

Reply:
i will correct what i found.